### DISCUSSION TITLE
How can I help this repository

### DISCUSSION THREAD

**Discussion by Lucs1590** [AlvaroCavalcante/auto_annotate#10] [General]
I love open source and this repository get my attention. How can I help?

**Comment by AlvaroCavalcante** [comment]
Thank you @Lucs1590 I'm glad that you want to help! I'll think in some way that you may help me! If you help me to write a better readme and add an image/icon that describes well this repository I'll be happy :), thanks!

**Reply by Lucs1590** [reply]
I will do it!
